erdpy/transactions.py

```diff
@@ -54,16 +49,16 @@ def receiver_username_decoded(self) -> str:
         return self._field_decoded("receiverUsername")
 
     def _field_encoded(self, field: str) -> str:
-        bytes = self.__dict__.get(field, None).encode("utf-8")
-        encoded = base64.b64encode(bytes).decode()
+        field_bytes = self.__dict__.get(field, None).encode("utf-8")
+        encoded = base64.b64encode(field_bytes).decode()
         return encoded
 
     def _field_decoded(self, field: str) -> str:
         return base64.b64decode(self.__dict__.get(field, None)).decode()
 
     def sign(self, account: Account):
```

Comment:
maybe use Union[Account, LedgerAccount] for the account?

Reply:
removed the Union as now Account is the base class of the LedgerAccount